api/v1alpha7/types.go

```diff
@@ -104,6 +104,20 @@ type SubnetFilter struct {
 	NotTagsAny      string `json:"notTagsAny,omitempty"`
 }
 
+type RouterFilter struct {
+	ID          string `json:"id,omitempty"`
+	Name        string `json:"name,omitempty"`
+	Description string `json:"description,omitempty"`
+	ProjectID   string `json:"projectId,omitempty"`
+	Limit       int    `json:"limit,omitempty"`
+	Marker      string `json:"marker,omitempty"`
+	SortKey     string `json:"sortKey,omitempty"`
```

Comment:
I think you missed these 3 (Limit-SortKey).

Reply:
Oh, yeah. I'll fix that now